Sources/XcodeGenKit/SpecGenerator.swift

```diff
@@ -587,7 +587,7 @@ private extension Scheme {
         self.init(
             name: scheme.name,
             build: Scheme.Build(
-                targets: [BuildTarget(target: TargetReference(name: buildableReference.blueprintName, location: .local))],
+                targets: [BuildTarget(target: TestableTargetReference(name: buildableReference.blueprintName, location: .local))],
```

Comment:
I'm trying to figure out the correct implementation of this section in a seprate branch. See this commit: https://github.com/skofgar/XcodeGen/commit/49a35a0ad72e6883025bc5aa5aad3a73b3f29fef